source/lmp/ascend_env.sh.in

```diff
@@ -0,0 +1,9 @@
+DEEPMD_ROOT=@CMAKE_INSTALL_PREFIX@
+TENSORFLOW_INCLUDE_DIRS="@TensorFlow_INCLUDE_DIRS@"
+TENSORFLOWFRAMEWORK_DIRS="@TensorFlow_LIBRARY_PATH@"
+TENSORFLOWFRAMEWORK_PATH="@TensorFlowFramework_LIBRARY@"
+TENSORFLOW_LIBRARY_PATH="@TensorFlow_LIBRARY@"
+
+NNP_INC=" -std=c++11 -D@prec_def@ @TTM_DEF@ -DLAMMPS_VERSION_NUMBER=@LAMMPS_VERSION_NUMBER@ -DHUAWEI_ASCEND -I$TENSORFLOW_INCLUDE_DIRS -I$DEEPMD_ROOT/include/ "
```

Comment:
`-std=c++11` is different from `env.sh.in`. Please confirm if it needs changing.

Reply:
Thanks! we have fixed it.